manager/integration/tests/test_migration.py

```diff
@@ -123,7 +123,64 @@ def test_migration_with_unscheduled_replica(clients, volume_name):  # NOQA
     11. Enable the scheduling for the node and wait for rebuilding complete.
     12. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)
+
+    # test_nodes[0] : node which running script
+    # test_nodes[1] : migrate target node
+    # test_nodes[2] : node to schedule on/off
```

Comment:
According to your test steps 6-9, the volume is being migrated from node[1] to node[0]. Can we update the description here, target becomes the node[0] and destination becomes node[1].

Reply:
Hello @khushboo-rancher, I updated the description, thank you